packages/backend/src/notifications.ts

```diff
@@ -91,13 +93,16 @@ export default () => {
                     error: 'Global notifications is off',
                 });
             } else {
-                // Adding a user's notification channel to the database
-                await req.app.locals.db.addUsersNotificationChannel(account, {
-                    type: notificationChannelType,
-                    config: {
-                        recipientValue: recipientValue,
-                    },
-                });
+                // add new notification channel & send OTP for verification
```

Comment:
I would only pass the DB object once to the function instead then accessing the method from within the functions.